docs/source/contributing/implementing_distribution.md

```diff
@@ -88,11 +88,11 @@ blah = BlahRV()
 Some important things to keep in mind:
 
 1. Everything inside the `rng_fn` method is pure Python code (as are the inputs) and should not make use of other `Aesara` symbolic ops. The random method should make use of the `rng` which is a NumPy {class}`~numpy.random.RandomState`, so that samples are reproducible.
-1. Non-default `RandomVariable` dimensions will end up in the `rng_fn` via the `size` kwarg. The `rng_fn` will have to take this into consideration for correct output. `size` is the specification used by NumPy and SciPy and works like PyMC `shape` for univariate distributions, but is different for multivariate distributions. For multivariate distributions the __`size` excludes the `ndim_supp` support dimensions__, whereas the __`shape` of the resulting `TensorVariabe` or `ndarray` includes the support dimensions__. This [discussion](https://github.com/numpy/numpy/issues/17669) may be helpful to get more context.
+1. Non-default `RandomVariable` dimensions will end up in the `rng_fn` via the `size` kwarg. The `rng_fn` will have to take this into consideration for correct output. `size` is the specification used by NumPy and SciPy and works like PyMC `shape` for univariate distributions, but is different for multivariate distributions. For multivariate distributions the __`size` excludes the `ndim_supp` support dimensions__, whereas the __`shape` of the resulting `TensorVariabe` or `ndarray` includes the support dimensions__. For more context check {doc}`The dimensionality notebook <pymc:dimensionality=>`.
```

Comment:
`` {doc}`The dimensionality notebook <pymc:dimensionality=>` `` should be `` {doc}`The dimensionality notebook <dimensionality>` ``. The `=` when defining a target is not part of the target but of the syntax:

```
(target)=
# Title
```

so that it "indicates" that referencing `target` is equal to referencing `# Title`. The `pymc:` prefix is used in **other** websites/repos to indicate the reference is in the pymc one instead of the local one. Therefore, we use that in `pymc-examples` to reference pymc docs but not on pymc. Similarly, in pymc-examples we reference other notebooks straight away whereas to reference example notebooks from here we use `nb:` prefix (prefixes are defined in `intersphinx_mapping` variable of `conf.py`)

Reply:
Opened follow up #6168